platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -644,8 +656,9 @@ private boolean isPemParsable(String type, String fileName) {
     // check mime types
     if (PKCS7_TYPE.equals(type) || CERT_TYPE.equals(type) || PEM_TYPE.equals(type)) {
       return true;
-      // check file extensions
-    } else if (StringUtils.endsWithIgnoreCase(fileName, ".crt")
+    }
+    // check file extensions
+    if (StringUtils.endsWithIgnoreCase(fileName, ".crt")
```

Comment:
✏️ Can simplify the `if(cond1 || cond2...) then return true` to `return (cond1 || cond2...)`